src/bootstrap/test.rs

```diff
@@ -1629,6 +1629,7 @@ impl Step for CrateNotDefault {
             .path("src/librustc_lsan")
             .path("src/librustc_msan")
             .path("src/librustc_tsan")
+            .path("src/librustc_hwasan")
```

Comment:
`CrateNotDefault` code can be safely removed as far as I can see. It is never used.

Reply:
Makes sense. I removed changes to this file.